Documentation/Performance/PerfGettingStarted.md

```diff
@@ -0,0 +1,79 @@
+The rest of the guide, however, is helpful in setting up symbols when diving into the CPU view, along with a concrete set of things
+that you can first try as you explore the WPA tools.
+
+### Other Resources
+
+As a platform built on Unity, all the other performance recommendations for building mixed reality experiences are relevant here:
+
+* [Performance recommendations for Unity](https://docs.microsoft.com/en-us/windows/mixed-reality/performance-recommendations-for-unity)
+* [Performance recommendations for immersive headset apps](https://docs.microsoft.com/en-us/windows/mixed-reality/performance-recommendations-for-immersive-headset-apps)
+* [Performance recommendations for HoloLens apps](https://docs.microsoft.com/en-us/windows/mixed-reality/performance-recommendations-for-hololens-apps)
+
+## Common Considerations
+
+### Use Single Pass Instanced Rendering 
+
+This recommendation is called out in 
+[other](https://docs.microsoft.com/en-us/windows/mixed-reality/performance-recommendations-for-immersive-headset-apps)
+resources, but is worth repeating here because enabling this is crucial for performant MRTK apps.
+For more information on what Single Pass Instanced Rendering is, check out the Unity docs on 
+[Single Pass Stereo rendering](https://docs.unity3d.com/Manual/SinglePassStereoRendering.html)
+and [Single Pass instanced rendering](https://docs.unity3d.com/Manual/SinglePassInstancing.html).
+
+This [Unity doc](https://docs.unity3d.com/Manual/SinglePassStereoRenderingHoloLens.html) describes how
+to turn on Single Pass Instanced Rendering. To give some context, it’s been observed in some cases that
+turning on this option has saved ~5ms per frame (and when there is a budget of 16.6ms per frame in order
+to achieve 60 FPS, this is clearly a massive improvement). 
+
+Note that in order to enable this rendering option, it's possible that you may need to update your shaders
+to support single pass instanced rendering. See the “Adding Single Pass Stereo rendering support to
+Shaders” section of this [resource](https://docs.unity3d.com/Manual/SinglePassStereoRendering.html)
+for more details. All the built-in MRTK shaders already support single pass instanced rendering.
+Other packages may not (for example, Text Mesh Pro shaders).
+
+## Visual Profiler 
+
+The MRTK provides a Visual Profiler, which provides information about the current FPS and memory usage.
+
+It is useful as a tool to identify overall areas for further investigation (for example, if you notice that
+looking at specific parts of your experience cause memory to spike or FPS to tank, it can be helpful to
+identify that overall part of your app to dig further into). However, it is recommended to turn the
+profiler off when doing deeper dives in performance (when you already know what areas aren’t performant),
+as the profiler itself adds overhead to the scene. Ultimately when your app ships, it won’t have this
```

Comment:
developers will need to turn the diagnostic system off in their profile before shipping. maybe link to the diags documentation or repeat the guidance from there?

Reply:
Yoyo left the same feedback - I can add this once it's all there.